CHANGELOG.md

```diff
@@ -4,11 +4,28 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.18.0] - 2022-03-02
+
+### Added
+
+- Reference key storage api ([#447](https://github.com/paritytech/subxt/pull/447))
```

Comment:
I'd definitely put this in `Changed`. E.g. my existing code would break where I have calls to storage with pass by value keys.

Reply:
That makes sense, will change right away